config/create-gke-manifest.js

```diff
@@ -1445,6 +1445,19 @@ async function applyPostgres() {
               ]
             }
           ],
+          initContainers: [
+            {
+              name: 'postgres-data-permission-fix',
+              image: 'busybox',
+              command: ['/bin/chmod', '-R', '777', '/bitnami/postgresql'],
```

Comment:
Never use 777 would be 755, (or if not 770,774 or 775) not be enough?

Reply:
Lets see, found this on their github issues :D

Reply:
Yes worked